crates/dpapi-transport/Cargo.toml

```diff
@@ -1,7 +1,13 @@
 [package]
 name = "dpapi-transport"
 version = "0.1.0"
-edition = "2024"
+edition = "2021"
```

Comment:
suggestion: Feel free to keep edition 2024 for new code. We’ll want to update to 2024 eventually.

Reply:
I just wanted to be consistent. I know that crates can have different edition versions, but my inner voice says that I want it to be the same for all crates in the workspace 😄